### ISSUE TITLE
Make drag and drop more user friendly

### ISSUE THREAD

**Issue by Celant** [symless/synergy#5793]
Problem: Drag and drop doesn't look like it's actually doing anything
Solution: Make drag and drop more user-friendly

**Comment by cpt-farron** [comment]
Yep, had this software since December and all I've wanted was for the D/D feature to work. But it doesn't at all. It was the main reason I bought it...

**Comment by Crytiqal** [comment]
I have never been able to drag and drop a file from server<->client.

Is this supposed to be a feature?
